src/test/java/org/sagebionetworks/bridge/udd/worker/BridgeUddWorkerTest.java

```diff
@@ -82,7 +86,7 @@ public void test() throws Exception {
         // mock Synapse packager
         SynapsePackager mockPackager = mock(SynapsePackager.class);
         when(mockPackager.packageSynapseData(same(mockSynapseToSchema), eq("test-health-code"),
-                any(BridgeUddRequest.class))).thenAnswer(invocation -> {
+                any(BridgeUddRequest.class), same(mockSurveyTableIdSet))).thenAnswer(invocation -> {
             // Second request is in March, that has no data. Third request is in August, that has data.
             BridgeUddRequest request = invocation.getArgumentAt(2, BridgeUddRequest.class);
             int startMonth = request.getStartDate().getMonthOfYear();
```

Comment:
You're dealing with Synapse, it deserves some integration tests to verify the Synapse behavior is as you expect it to be. What's your plan for this?